qutip/tests/test_wigner.py

```diff
@@ -36,11 +36,181 @@
 from numpy.random import rand
 from numpy.testing import assert_, run_module_suite, assert_equal
 
-from qutip.states import coherent, fock
-from qutip.wigner import wigner
+from qutip.states import coherent, fock, ket, bell_state
+from qutip.wigner import wigner, wigner_transformation, _parity
 from qutip.random_objects import rand_dm, rand_ket
 
 
+def test_wigner_bell1_su2parity():
+    """wigner: testing the SU2 parity of the first Bell state.
+    """
+    psi = bell_state('00')
+
+    steps = 100
```

Comment:
Could you set all those steps to 20 are 25, using 100 make clean images but it takes a lot of time and not needed for tests.

Reply:
Done

> It seems good and could be accepted, but I would prefer it if you could make the tests faster.
> Maybe have a better description so that users knows when to use `wigner_transformation` and when to use `wigner`. Both having only a 1 line description, it can be confusing.
> 
> (Sorry for the delay.)

I made the changes you requested. Are we ready for merge?